README.md

```diff
@@ -55,7 +55,7 @@ See [config.example.json](./integritybackend/config.example.json) for an example
 See [config.py](./integritybackend/config.py) for the available variables and some notes about each. In development, you can use a local `.env` file setting environment variables. See `.env.example` for an example.
 
 Most importantly, you will need to provide:
-* `CLAIM_TOOL_PATH`: A path to a fully working `claim_tool` binary. The server should have permissions to execute it, and it should be correctly configured with its keys.
+* `C2PATOOL`: A path to a v0.3.0 `c2patool` binary. The server should have permissions to execute it, and it should be correctly configured with its keys. You can download one from TODO and the repo is [here](https://github.com/contentauth/c2patool).
```

Comment:
`TODO`? Did we decide where to put the builds?

Reply:
Fixed. Adobe makes their own builds now, so I've just linked to those. Note there's no Raspberry Pi ones yet.